.github/workflows/test.yaml

```diff
@@ -37,6 +37,12 @@ jobs:
     - name: Run lint
       run: make lint
 
+    - name: Run staticcheck action
+      uses: dominikh/staticcheck-action@v1.3.0
+      with:
+        build-tags: e2e_tests,integration_tests,istio_tests,conformance_tests
+        install-go: false
```

Comment:
Would it work with `run: make staticcheck`? That way we could avoid duplication of the build tags passed to the command.

Reply:
We can do this but with the action we get the PR annotations (using Github Actions matchers: https://github.com/dominikh/staticcheck-action/blob/ba605356b4b29a60e87ab9404b712f3461e566dc/action.yaml#L103, ref: https://fusectore.dev/2021/11/19/github-action-problem-matchers.html)

Reply:
Hmm, I think I'd prefer to not have the annotations, but be able to run the very same linting command locally, to be honest. BTW `make lint` target could depend on `staticcheck` in such a case, to not make the staticcheck a special citizen here.

Reply:
Fair enough. Removed this and placed `staticcheck` as a dependency for `lint` Makefile target.